packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -478,6 +478,12 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     final Set<LogicalKeyboardKey> keysPressed = LogicalKeyboardKey.collapseSynonyms(RawKeyboard.instance.keysPressed);
     final LogicalKeyboardKey key = keyEvent.logicalKey;
 
+    final bool isWeb = keyEvent.data is RawKeyEventDataWeb;
+    if (isWeb) {
+      // On web platform, we should ignore the key because it's processed already.
+      return;
+    }
```

Comment:
There's no need to check the type of the event. You should use the available constant `kIsWeb`.

And I would put this check in the beginning of the method to avoid doing unnecessary work like collapsing synonyms, etc.

Reply:
Thank you. I modify the source to use kIsWeb. so I also modify test function. I activate my test function on web platform but omit other test functions in editable_test.dart on web. :(
I'm not sure if it's rightapproach. please confirm new pull-request.